integrations/gitlab/gitlab_integration/gitlab_service.py

```diff
@@ -34,6 +34,15 @@
 
 
 class GitlabService:
+    all_events_in_webhook: list[str] = ["push_events", 
+                                        "merge_requests_events", 
+                                        "issues_events", 
+                                        "job_events", 
+                                        "pipeline_events", 
+                                        "releases_events", 
+                                        "tag_push_events", 
+                                        "subgroup_events", 
+                                        "confidential_issues_events"]
```

Comment:
interesting why we are listening by default to  `confidential_issues_events` 🤔 
@yairsimantov20 can you elaborate if this is still needed?